### ISSUE TITLE
MySQL support

### ISSUE THREAD

**Issue by chronic414** [cadon/ARKStatsExtractor#689]
Hi there,

it would be nice if you could implement a connector to store the data in a MySQL database. This would really helpfull for tribes who wants to work at the same time with the tool. I know that you could store the XML file on a network share. The problem with this is to find a hoster that allows multiple people with different accounts to access the file via WebDAV.

**Comment by cadon** [comment]
MySQL or similar is on the todo-list without promise it will be done.

Did you try googledrive or dropbox yet?

**Comment by chronic414** [comment]
Yes, in Google Drive we had sync timing issues. With Dropbox we have to either pay monthly to use it with multiple accounts or to share one account. But then we cant control who have access. And in both cases you have to install 3rd party software.

**Comment by cadon** [comment]
Dropbox basic is free. Most people use dropbox for sharing the library, afaik that works well. Googledrive had some issues with saving, these got hopefully fixed around two months ago, if you haven't tried it since then, give it another try.
